### PR TITLE
[internal] Add RemoveAfter property to additional testing artifacts

### PR DESCRIPTION
**Pull request by pritishpai**
## Changes
Add RemoveAfter property to additional artifacts

Add it as property to:

tables
schemas

### Linked issues
Party introduces #977 

### Functionality 

- [ ] added relevant user documentation
- [ ] added new CLI command
- [ ] modified existing command: `databricks labs ucx ...`
- [ ] added a new workflow
- [ ] modified existing workflow: `...`
- [ ] added a new table
- [ ] modified existing table: `...`

### Tests

- [x] manually tested
- [ ] added unit tests
- [x] added integration tests
- [ ] verified on staging environment (screenshot attached)

### CODE REVIEW

File: src/databricks/labs/ucx/mixins/fixtures.py [hunk 1]
```diff
@@ -975,7 +975,10 @@ def create(*, catalog_name: str = "hive_metastore", name: str | None = None) ->
             name = f"ucx_S{make_random(4)}".lower()
         full_name = f"{catalog_name}.{name}".lower()
         sql_backend.execute(f"CREATE SCHEMA {full_name}")
-        schema_info = SchemaInfo(catalog_name=catalog_name, name=name, full_name=full_name)
+        remove_after_property = {"RemoveAfter": get_test_purge_time()}
```
Comment (nfx) [inline src/databricks/labs/ucx/mixins/fixtures.py:978]: This doesn't have an effect - add it to sql statement

File: src/databricks/labs/ucx/mixins/fixtures.py [hunk 2]
```diff
@@ -1073,6 +1088,11 @@ def create(  # pylint: disable=too-many-locals,too-many-arguments,too-many-state
             storage_location = storage_override
 
         sql_backend.execute(ddl)
+
+        # CTAS AND NON_DELTA does not support TBLPROPERTIES
```
Comment (nfx) [inline src/databricks/labs/ucx/mixins/fixtures.py:1092]: Really? Why?

**Review by nfx — CHANGES_REQUESTED** [review]

File: src/databricks/labs/ucx/mixins/fixtures.py (hunk 2 above)
Comment (pritishpai) [reply src/databricks/labs/ucx/mixins/fixtures.py:1091]: image

Comment (pritishpai) [reply src/databricks/labs/ucx/mixins/fixtures.py:1091]: It is failing with SYNTAX error.
image

Comment (nfx) [reply src/databricks/labs/ucx/mixins/fixtures.py:1091]: Can it be done in a second SQL query?

File: src/databricks/labs/ucx/mixins/fixtures.py [hunk 3]
```diff
@@ -1073,6 +1087,11 @@ def create(  # pylint: disable=too-many-locals,too-many-arguments,too-many-state
...
+        if ctas or non_delta:
+            sql_backend.execute(ctas_tbl_properties_ddl)
```
Comment (nfx) [inline src/databricks/labs/ucx/mixins/fixtures.py:1093]: Execute a second sql query that sets table properties

**Review by nfx — CHANGES_REQUESTED** [review]

File: src/databricks/labs/ucx/mixins/fixtures.py (hunk 3 above)
Comment (pritishpai) [reply src/databricks/labs/ucx/mixins/fixtures.py:1093]: Yup the query being executed here 'ctas_tbl_properties_ddl' is the second ALTER TABLE query for setting table properties

Comment (pritishpai) [reply src/databricks/labs/ucx/mixins/fixtures.py]: Renamed it to 'alter_table_tbl_properties_ddl'

**Review by nfx — APPROVED** [review]
lgtm

**Pull request merged**